lighthouse/datadog_checks/lighthouse/data/conf.yaml.example

```diff
@@ -0,0 +1,26 @@
+init_config:
```

Comment:
We try to follow a single format for all configuration file described [here](https://docs.datadoghq.com/developers/integrations/new_check_howto/#configuration). As you can see the testing pipeline now fails to validate this configuration file while it was working before. This is due to caching on our side, the tests were using an old version of the configuration validator which was more permissive.  Here is a suggested fix for it:

```
init_config:

instances:
    ## @param min_collection_interval - integer - required
    ## The lighthouse check can be resource intensive
    ## and can take ~10 seconds per instance, this has only been tested
    ## with a min collection interval of 60 seconds for a few instances
    #
  - min_collection_interval: 60

    ## @param url - string - required
    ## url of the website to get lighthouse metrics of
    ## ex: "https://www.datadoghq.com"
    #
    url: <URL_PLACEHOLDER>

    ## @param name - string - required
    ## name of your website.
    ## ex: "datadog_homepage"
    #
    name: <NAME_PLACEHOLDER>

    ## @param tags - list of key:value elements - optional
    ## List of tags to attach to every metric, event and service check emitted by this integration.
    ##
    ## Learn more about tagging: https://docs.datadoghq.com/tagging/
    #
    # tags:
    #   - <KEY_1>:<VALUE_1>
    #   - <KEY_2>:<VALUE_2>
```

Reply:
thank you for this, apologies on the formatting originally, cleaned up and updated